wiki/People/The_Team/Global_Moderation_Team/en.md

```diff
@@ -16,6 +16,7 @@ The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all o
 
 *Note: All GMT members speak English in addition to the language(s) listed below unless noted otherwise. For a list of moderators grouped by the languages moderated, see [Language Moderators](Language_Moderators).*
```

Comment:
```suggestion
*Note: All GMT members speak English in addition to the language(s) listed below unless noted otherwise. For a list of moderators grouped by the languages moderated, see [Language Moderators](#grouped-by-languages-moderated).*
```
or remove instead?

Reply:
woops, I think i meant to remove it